### PR TITLE
fix(lib): fix comments

### PR DESCRIPTION
**Pull request by va-stefanek**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

There were errors from eslint.  We have not control over way of updating storage.

Issue Number: #5 

## What is the new behavior?

Eslint issues are fixed. We can provide our custom callback function to update storage.

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/ngneat/storage/src/lib/utils.ts [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
   }
 
   return of(value);
 }
 
-export function getFromStorage<T = any>(key: string, persistManager: PersistManager<T>): Observable<T> {
+export function getFromStorage<T = unknown>(
+  key: string,
+  persistManager: PersistManager<T>
+): Observable<T> {
```
Comment (NetanelBasal) [inline projects/ngneat/storage/src/lib/utils.ts:22]: To be honest, I'd just go with one class the takes the persistManager once and exposes the methods:

```ts
class StorageFacade<T> {
  constructor(private manager: PersistManager<T>) {}

  update() {}

  get() {}

   ....
}
```

Comment (va-stefanek) [reply projects/ngneat/storage/src/lib/utils.ts:22]: Ok will do that this weekend :+1:

Comment (va-stefanek) [reply projects/ngneat/storage/src/lib/utils.ts]: Done

**Comment by NetanelBasal** [comment]
You have errors in CI - https://github.com/ngneat/storage/pull/6/checks?check_run_id=2814540463

**Pull request merged**